.github/workflows/label_stale_issues.yml

```diff
@@ -0,0 +1,24 @@
+# Labels all issues with no activity for 14 days as "stale"
+# Runs everday at 00:00
+#-------------------------------------------------------------
+
+name: Check stale
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          include-only-assigned: true
+          days-before-issue-stale: 14
+          days-before-close: -1
+          stale-issue-label: "stale"
```

Comment:
Let's not add "stale" label to the issue. Instead, is it possible to just unassign?

Reply:
I think not leaving a label should be possible with the current GitHub action, right?